vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -1651,14 +1651,6 @@ public boolean onCreateOptionsMenu(Menu menu) {
             mSearchInRoomMenuItem = menu.findItem(R.id.ic_action_search_in_room);
             mUseMatrixAppsMenuItem = null; //will be reconnected later menu.findItem(R.id.ic_action_matrix_apps);
 
-            if (null != mRoom) {
```

Comment:
Do not remove this code, it has been introduced by https://github.com/vector-im/riot-android/issues/1856 in order to inform correctly the user that he has been kicked or banned.

Reply:
When I click on a room from which I was excluded (kicked or banned), the roomPreview opens and we pass in the code above.
The crash results from a NullPointerException on the following object: menu.findItem(R.id.ic_action_room_leave)

<img width="331" alt="capture d ecran 2018-07-19 a 10 08 56" src="https://user-images.githubusercontent.com/36439413/42930115-6aa7263e-8b3c-11e8-8054-287f74722089.png">

In order to fix the crash, I can add the following test "null != menu.findItem((R.id.ic_action_room_leave)". In this case, the result is the same as when the above code block is deleted. I don't understand the point of keeping this code.

<img width="671" alt="capture d ecran 2018-07-19 a 10 19 25" src="https://user-images.githubusercontent.com/36439413/42930485-695003d6-8b3d-11e8-862e-ffc85e876886.png">